borg/archive.py

```diff
@@ -12,9 +12,10 @@
 import time
 from io import BytesIO
 from . import xattr
-from .platform import acl_get, acl_set
-from .chunker import Chunker
-from .hashindex import ChunkIndex
+if not os.environ.get('BORG_GEN_USAGE', False):
```

Comment:
i moved this to `helpers.dected_cython()` to make it more generic and potentially useful in the future - for, say, tests of non-cython alternatives if they are available.